src/components/dialogs/contingency-list/criteria-based/criteria-based-form.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { filterEquipmentDefinition } from '../../../../utils/equipment-types';
+import {
+    COUNTRIES_1,
+    COUNTRIES_2,
+    EQUIPMENT_TYPE,
+    NOMINAL_VOLTAGE_1,
+    NOMINAL_VOLTAGE_2,
+} from '../../../utils/field-constants';
+import React, { useMemo } from 'react';
+import { useWatch } from 'react-hook-form';
+import { gridItem } from '../../../utils/dialog-utils';
+import { Grid } from '@mui/material';
+import CountriesInput from '../../../utils/countries-input';
+import RangeInput from '../../../utils/range-input';
+import MuiSelectInput from '../../../utils/mui-select-input';
+import { CONTINGENCY_LIST_EQUIPMENTS } from '../contingency-list-utils';
+
+const CriteriaBasedForm = () => {
+    const watchEquipmentType = useWatch({
+        name: EQUIPMENT_TYPE,
+    });
+
+    const equipmentTypeSelectionField = (
+        <MuiSelectInput
+            name={EQUIPMENT_TYPE}
+            options={Object.values(CONTINGENCY_LIST_EQUIPMENTS)}
+            label={'equipmentType'}
+        />
+    );
+
+    const isLineOrHvdc = useMemo(
+        () =>
+            watchEquipmentType === filterEquipmentDefinition.LINE.type ||
+            watchEquipmentType === filterEquipmentDefinition.HVDC_LINE.type,
+        [watchEquipmentType]
+    );
+
+    const isLineOrTransformer = useMemo(
+        () =>
+            watchEquipmentType === filterEquipmentDefinition.LINE.type ||
+            watchEquipmentType ===
+                filterEquipmentDefinition.TWO_WINDINGS_TRANSFORMER.type,
+        [watchEquipmentType]
+    );
+
+    const countries1 = (
+        <CountriesInput name={COUNTRIES_1} titleMessage={'Countries1'} />
```

Comment:
So if there is only one country field, it's called "Countries1" ? It should be "Countries" no ?